compiler/plugins/target/AMD-AIE/aie/AMDAIECreatePathFindFlows.cpp

```diff
@@ -573,22 +158,25 @@ LogicalResult DynamicTileAnalysis::runAnalysis(DeviceOp &device) {
     TileOp dstTile = cast<TileOp>(flowOp.getDest().getDefiningOp());
     TileLoc srcCoords = {srcTile.colIndex(), srcTile.rowIndex()};
     TileLoc dstCoords = {dstTile.colIndex(), dstTile.rowIndex()};
-    Port srcPort = {flowOp.getSourceBundle(), flowOp.getSourceChannel()};
-    Port dstPort = {flowOp.getDestBundle(), flowOp.getDestChannel()};
-    LLVM_DEBUG(llvm::dbgs()
-               << "\tAdding Flow: (" << srcCoords.col << ", " << srcCoords.row
-               << ")" << stringifyWireBundle(srcPort.bundle) << srcPort.channel
-               << " -> (" << dstCoords.col << ", " << dstCoords.row << ")"
-               << stringifyWireBundle(dstPort.bundle) << dstPort.channel
-               << "\n");
+    Port srcPort = {toStrmT(flowOp.getSourceBundle()),
+                    flowOp.getSourceChannel()};
+    Port dstPort = {toStrmT(flowOp.getDestBundle()), flowOp.getDestChannel()};
     pathfinder->addFlow(srcCoords, srcPort, dstCoords, dstPort);
   }
 
   // add existing connections so Pathfinder knows which resources are
   // available search all existing SwitchBoxOps for exising connections
   for (SwitchboxOp switchboxOp : device.getOps<SwitchboxOp>()) {
     for (ConnectOp connectOp : switchboxOp.getOps<ConnectOp>()) {
-      if (!pathfinder->addFixedConnection(connectOp))
+      auto sb = connectOp->getParentOfType<SwitchboxOp>();
+      // TODO: keep track of capacity?
```

Comment:
```suggestion
      // Don't add handwritten connections for Shim NOC tiles.
      // TODO: keep track of capacity so that even if we don't try to add those
      // connections (as actual routes) we might consider checking if capacity 
      // has been exceeded.
```